src/ds/tree/insertTreeNode.js

```diff
@@ -14,7 +14,7 @@
  *
  * @example <caption>How to use</caption>
  *
- * const node = insertTreeNode({ parent, node, offset })
+ * insertTreeNode({ parent, node, offset })
```

Comment:
revert this change please. It is not relevant to this PR. And, it has been fixed elsewhere.